torch/_torch_docs.py

```diff
@@ -1465,6 +1465,47 @@ def merge_dicts(*dicts):
     tensor([ 0.5000, -0.4702, -0.4599,  0.5000])
 """.format(**common_args))
 
+add_docstr(torch.complex,
+           r"""
+complex(real, imag, out=None) -> Tensor
+
+Constructs a complex-valued tensor from Cartesian coordinates.
+
+:attr:`real` and :attr:`imag` should be floating-point tensors of the same
+dtype. ``torch.float32`` tensors will lead to an output of ``torch.complex64``,
+and ``torch.float64`` tensors will lead to an output of ``torch.complex128``.
+
+Example::
+    >>> real = torch.tensor([1, 2], dtype=torch.float32)
+    >>> imag = torch.tensor([3, 4], dtype=torch.float32)
+    >>> z = torch.complex(real, imag)
+    >>> z
+    tensor([(1.+3.j), (2.+4.j)])
+    >>> z.dtype
+    torch.complex64
+
+""".format(**common_args))
+
+add_docstr(torch.complex_polar,
+           r"""
+complex_polar(abs, angle, out=None) -> Tensor
+
+Constructs a complex-valued tensor from polar coordinates.
```

Comment:
Maybe something like:

Constructs a complex tensor whose elements are Cartesian coordinates corresponding to the polar coordinates with magnitude magnitude :attr:`abs` and angle :attr:`angle`. 

Follow-up question: should "abs" be "abs" or something like "magnitude?" cc @anjali411 to review naming. 

You could even show the math for this one, if you want to write some tex.

Reply:
Sure. About the naming, I think abs/angle is what's used elsewhere in the code right now (I see it in zmath.h and vec256_base.h). An alternative is mag/phase, which I guess removes the conflict with Python's built in abs.

Reply:
In those other contexts I think "abs" stands for absolute value, which is not here, it's just that usually in polar coordinates we don't think of the "distance" or "magnitude" term as being signed since it plus the angle are a vector.